decidim-core/config/locales/en.yml

```diff
@@ -1498,6 +1499,26 @@ en:
           alert: We have updated our Terms of Service, please review them.
           body: Please take a moment to review updates to our Terms of Services. Otherwise you will not be able to use the platform.
           title: 'Required: Review updates to our terms of service'
+    participants_account_mailer:
+      inactivity_notification:
+        body: |
+          Our records indicate you created a %{organization_name} account on %{date_account_creation} and have not been active since %{date_last_connection}.
+          To protect your privacy, this account has been scheduled for automatic deletion in %{days} days.
+          If you wish to keep your %{organization_name} account, simply %{login_url} before this time, and your account will be preserved.
+        greetings: |
+          Kind regards,
+          %{organization_name}
```

Comment:
```suggestion
        body_html: |
          Our records indicate you created a %{organization_name} account on %{date_account_creation} and have not been active since %{date_last_connection}.
          To protect your privacy, this account has been scheduled for automatic deletion in %{days} days.
          If you wish to keep your %{organization_name} account, simply %{login_url} before this time, and your account will be preserved.
        greetings_html: |
          Kind regards,
          %{organization_name}
```